### ISSUE TITLE
[Docs]: Confusion around Validator Directives vs. Service Validation

### ISSUE THREAD

**Issue by JoshFerge** [redwoodjs/redwood#8996]
### Summary and description

my docs version: Redwood 6.0

I'd like to have my services validate that the logged in user can only edit the data that they own. I'm trying to figure out in the docs the most ergonomic way to do this. 

On this page: https://redwoodjs.com/docs/directives#when-should-i-use-a-redwood-directive
<img width="580" alt="Screenshot 2023-07-30 at 5 40 04 PM" src="https://github.com/redwoodjs/redwood/assets/1976777/1c11d376-71f5-48fb-8aca-515e5f48af4e">

the docs say to do the check in your service instead.

On the service docs: https://redwoodjs.com/docs/services#service-validations  it says you should do this in your validator directive. Is there a contradiction here, or what am I missing?
<img width="784" alt="Screenshot 2023-07-30 at 5 40 24 PM" src="https://github.com/redwoodjs/redwood/assets/1976777/07d072a2-850d-4762-ba3e-1fa38749ea1a">

### Are you interested in working on this?

- [X] I'm interested in working on this

**Comment by Tobbe** [comment]
I agree. That does sound contradictory 🧐 I *think* the Service Validations docs are wrong, and personally I'd implement the check you're asking about inside my services. Especially as you then also get the check whenever you call your services from other places and not just directly from graphql. 

But @dthyresson is our resident expert on these things, so hopefully he can provide some guidance here 🗺️
